src/speculators/convert/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Checkpoint conversion utilities for Speculators.
+
+This module provides tools to convert existing speculator checkpoints
+(Eagle, HASS, etc.) into the standardized speculators format.
+"""
+
+from speculators.convert.eagle.eagle_converter import EagleConverter
+
+__all__ = ["EagleConverter"]
```

Comment:
nit: exporting `speculators.convert.EagleConverter` and `speculators.convert.eagle.EagleConverter` feels awkward to old people like me. I think exporting them all in `speculators.convert` is clean and in-line with how imports are in `transformers` and `torch`